modules/xml/src/test/java/org/mule/util/XmlLibrariesVerificationTestCase.java

```diff
@@ -58,4 +59,11 @@ public void transformer()
         TransformerFactory factory = TransformerFactory.newInstance();
         assertThat(factory, instanceOf(org.apache.xalan.processor.TransformerFactoryImpl.class));
     }
+
+    @Test
+    public void schema()
```

Comment:
can you confirm the Validator implementation won't change based on how the classpath is fromed?